### DISCUSSION TITLE
我们能不能把镜像放到一个国内源?

### DISCUSSION THREAD

**Discussion by chenliu1993** [RayWangQvQ/BiliBiliToolPro#310] [Ideas]
我理解镜像放到了docker registry上，然而docker现在拉取镜像很慢，经常拉取失败导致负载启动不起来，我们能把他放到一个国内镜像网站嘛？类似aliyun或者github自带的ghcr.io这类的？

另外，简单添加了Chart后，在我的harbor上扫描了一下，有一些严重的CVE，这样在如果放在集群里运行会不会有一些安全问题？

**Comment by catlair** [comment]
docker 国内会主动同步啊

**Comment by RayWangQvQ** [comment]
docker国内应该也不慢吧？

后面会放ghcr.io，抽空我来写个github actions。

**Comment by chenliu1993** [comment]
是我服务器带宽太低了，提高后好了不少

**Comment by RayWangQvQ** [comment]
ghcr.io有了
